<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GS232ControllerGUI</class>
 <widget class="RollupWidget" name="GS232ControllerGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>231</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>GS-232 Rotator Controller</string>
  </property>
  <widget class="QWidget" name="settingsContainer" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>211</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="controlLayout">
      <item>
       <widget class="ButtonSwitch" name="startStop">
        <property name="toolTip">
         <string>Start/stop controller</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/play.png</normaloff>
          <normalon>:/stop.png</normalon>:/play.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="azimuthtLabel">
        <property name="text">
         <string>Azimuth</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="azimuth">
        <property name="toolTip">
         <string>Target azimuth in degrees</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>450.000000000000000</double>
        </property>
        <property name="value">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="azimuthCurrentText">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Current azimuth in degrees</string>
        </property>
        <property name="text">
         <string>360.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="elevationLabel">
        <property name="text">
         <string>Elevation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="elevation">
        <property name="toolTip">
         <string>Target elevation in degrees</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="value">
         <double>180.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="elevationCurrentText">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Current elevation in degrees</string>
        </property>
        <property name="text">
         <string>180.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="2">
       <widget class="QLabel" name="toleranceLabel">
        <property name="text">
         <string>Tolerance</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="azimuthOffsetLabel">
        <property name="text">
         <string>Azimuth offset</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="azimuthOffset">
        <property name="toolTip">
         <string>Specify an offset angel in degrees that will be added to the target azimuth to correct for misalignment</string>
        </property>
        <property name="minimum">
         <number>-360</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QSpinBox" name="elevationOffset">
        <property name="toolTip">
         <string>Specify an offset angle in degrees that will be added to the target elevation to correct for misalignment</string>
        </property>
        <property name="minimum">
         <number>-180</number>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="serialPort">
        <property name="toolTip">
         <string>Name of serial port to use to connect to the GS-232 controller</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QComboBox" name="baudRate">
        <property name="toolTip">
         <string>Serial port baud rate for the GS-232 controller</string>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>1200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>230400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>460800</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="serialLabel">
        <property name="text">
         <string>Serial Port</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="elevationOffsetLabel">
        <property name="text">
         <string>Elevation offset</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="elevationMaxLabel">
        <property name="text">
         <string>Elevation max</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="elevationMin">
        <property name="maximum">
         <number>180</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="protocol">
        <property name="toolTip">
         <string>Command protocol</string>
        </property>
        <item>
         <property name="text">
          <string>GS-232</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SPID</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="azimuthMinLabel">
        <property name="text">
         <string>Azimuth min</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QSpinBox" name="azimuthMax">
        <property name="maximum">
         <number>450</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="tolerance">
        <property name="toolTip">
         <string>Tolerance in degrees</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="protocolLabel">
        <property name="text">
         <string>Protocol</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QSpinBox" name="elevationMax">
        <property name="maximum">
         <number>180</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="elevationMinLabel">
        <property name="text">
         <string>Elevation min</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="baudRateLabel">
        <property name="text">
         <string>Baud rate</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="azimuthMin">
        <property name="maximum">
         <number>450</number>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="azimuthMaxLabel">
        <property name="text">
         <string>Azimuth max</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="track">
        <property name="toolTip">
         <string>Check to enable automatic tracking of azimuth and elevation from the specified channel</string>
        </property>
        <property name="text">
         <string>Track</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QLineEdit" name="targetName">
        <property name="toolTip">
         <string>Name of the target being tracked as indicated by the source channel / feature</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="targetNameLabel">
        <property name="text">
         <string>Target</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="targetsLabel">
        <property name="text">
         <string>Source</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QComboBox" name="targets">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Target to track</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RollupWidget</class>
   <extends>QWidget</extends>
   <header>gui/rollupwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>startStop</tabstop>
  <tabstop>azimuth</tabstop>
  <tabstop>elevation</tabstop>
  <tabstop>serialPort</tabstop>
  <tabstop>baudRate</tabstop>
  <tabstop>azimuthOffset</tabstop>
  <tabstop>elevationOffset</tabstop>
  <tabstop>azimuthMin</tabstop>
  <tabstop>azimuthMax</tabstop>
  <tabstop>elevationMin</tabstop>
  <tabstop>elevationMax</tabstop>
 </tabstops>
 <resources>
  <include location="../../../sdrgui/resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
